### ISSUE TITLE
Cannot find module Proptypes

### ISSUE THREAD

**Issue by againer** [anthonyjgrove/react-google-login#78]
```
{ [Error: ./~/react-google-login/dist/google-login.js

Module not found: Error: Cannot resolve module 'PropTypes'
```

Just started happening about an hour ago for my builds.  It seems like something in 2.9.0 is broken now.  I think it might be related to:
https://github.com/anthonyjgrove/react-google-login/commit/98f9b0d27142f7e3134315bfdc2ce7fa7086d342

**Comment by againer** [comment]
Looks like maybe you forgot to run bundle when you cut the release.

**Comment by Coobers** [comment]
Production blocker...

**Comment by anthonyjgrove** [comment]
Should be fixed with 2.9.1.
